skimage/feature/peak.py

```diff
@@ -159,3 +159,73 @@ def peak_local_max(image, min_distance=10, threshold_abs=0, threshold_rel=0.1,
         nd_indices = tuple(coordinates.T)
         out[nd_indices] = True
         return out
+
+
+def get_scale_local_maximas(cube_coordinates, laplacian_cube):
```

Comment:
This can be sped up by a Cython implementation since a majority of it's work is looping through numbers.
You can also argue that in case of `_prune_blobs` ( which I wrote myself )